commit dbd47622eb99fefb3538a22baec3def002aa56f5
Author: Jia Tan <jiat0218@gmail.com>
Date:   Thu Jan 26 23:17:41 2023 +0800

    liblzma: Fix typo in documentation in container.h
    
    lzma_microlzma_decoder -> lzma_microlzma_encoder

diff --git a/src/liblzma/api/lzma/container.h b/src/liblzma/api/lzma/container.h
index b47da05e..088b63a9 100644
--- a/src/liblzma/api/lzma/container.h
+++ b/src/liblzma/api/lzma/container.h
@@ -925,7 +925,7 @@ extern LZMA_API(lzma_ret) lzma_stream_buffer_decode(
 /**
  * \brief       MicroLZMA decoder
  *
- * See lzma_microlzma_decoder() for more information.
+ * See lzma_microlzma_encoder() for more information.
  *
  * The lzma_code() usage with this decoder is completely normal. The
  * special behavior of lzma_code() applies to lzma_microlzma_encoder() only.